integrations/tensorflow/test/lit.cfg.py

```diff
@@ -44,21 +46,26 @@
     'imagenet_test_data.py',
 ]
 
+PYTHON_EXEC = sys.executable
+
+if PYTHON_EXEC is None:
```

Comment:
Add a comment: "# Some environments use a version of Python built against an embedded interpreter and lack a sys.executable. In this case, we allow an explicit override from the environment."